### DISCUSSION TITLE
Nitpicks from 2022 Playthough

### DISCUSSION THREAD

**Discussion by KnightCaptainU7** [exult/exult#267] [General]
This is a followup on the Phorum thread here: https://exult.sourceforge.io/forum/viewtopic.php?f=1&t=1791299

I've fixed a few issues tonight:
The Fawn Trial start was hanging the game, commented out getTrialNPC re-implementation to resolve.
The Furnace arena's two eggs were not working as expected, also commented them out.
Fixed the Ophidian Book code to not teleport the party unexpectedly. Seems originally the game was designed to have you reading the book lead to a teleportation. The variable for it is never set.

**Comment by KnightCaptainU7** [comment]
Ensorcio / Stefano owning the same mansion is also listed in the Moonshade hand-drawn design:
http://home.insightbb.com/~chilliedog/Htm/ultima_7_part_2.htm

**Comment by KnightCaptainU7** [comment]
In the pre-Batlin battle inside Spinebreaker, the henchmen are pretty weak and not armored. Palos and Brunt are unarmed completely, and Deadeye has only a two-handed sword.

**Comment by KnightCaptainU7** [comment]
The Infinity Bow, a magical weapon, is weaker than even the standard bow, dealing 3 damage. The standard bow does 6.

**Reply by r-h-o-n-a** [reply]
Remember the Infinity Bow does explosion damage in addition to its base damage, though, and to multiple targets. I just tried it out in-game and in practice 6 or 7 damage to a single target with an additional 2-4 to peripheral enemies wasn't rare.

**Comment by KnightCaptainU7** [comment]
In the latest playtest of my mod, I've removed all showstopper bugs.
